hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestEndpoints.java

```diff
@@ -458,7 +458,6 @@ public void testGetDatanodes() throws Exception {
               .findFirst().orElse(null);
       return (datanodeMetadata1 != null &&
           datanodeMetadata1.getContainers() == 1 &&
-          datanodeMetadata1.getOpenContainers() == 1 &&
```

Comment:
why delete this?

Reply:
I think open container has been tested by TestOpenContainerCount

Reply:
Original file doesn't have this line. I added it back when I didn't figure out a way to write test case for multiple containers

Reply:
IMO this can serve as another check, why not just keep it?

Reply:
sure